### PR TITLE
Add friend counter + Change alignment of player count text

### LINKED ISSUE #5911: Add a "counter" above the Friend heart in Server list
it would be nice if there was a small number above the friend heart that says how many friends are in this one server
(see screenshot for reference)
DDNet_wnGq83UXPv

### PR DESCRIPTION
**Pull request by l-ouis**
Added friend counter, changed the alignment of player count indicator to better accommodate friend counter + line up with the "Players" text at the top of the tab

image

Fixes #5911

## Checklist

- [x] Tested the change ingame
- [x] Provided screenshots if it is a visual change
- [ ] Tested in combination with possibly related configuration options
- [ ] Written a unit test (especially base/) or added coverage to integration test
- [ ] Considered possible null pointers and out of bounds array indexing
- [x] Changed no physics that affect existing maps
- [ ] Tested the change with [ASan+UBSan or valgrind's memcheck](https://github.com/ddnet/ddnet/#using-addresssanitizer--undefinedbehavioursanitizer-or-valgrinds-memcheck) (optional)

### CODE REVIEW

File: src/game/client/components/menus_browser.cpp [hunk 1]
```diff
@@ -420,20 +422,29 @@ void CMenus::RenderServerbrowserServerList(CUIRect View)
...
+						TextRender()->TextColor(1.0f, 1.0f, 1.0f, 1);
+					}
 				}
 
 				str_format(aTemp, sizeof(aTemp), "%i/%i", pItem->m_NumFilteredPlayers, ServerBrowser()->Max(*pItem));
 				if(g_Config.m_BrFilterString[0] && (pItem->m_QuickSearchHit & IServerBrowser::QUICK_PLAYER))
 					TextRender()->TextColor(0.4f, 0.4f, 1.0f, 1);
 				float FontSize = 12.0f;
-				UI()->DoLabelStreamed(*pItem->m_pUIElement->Rect(gs_OffsetColPlayers), &Button, aTemp, FontSize, TEXTALIGN_RIGHT, -1, 1, false);
+				UI()->DoLabelStreamed(*pItem->m_pUIElement->Rect(gs_OffsetColPlayers), &Button, aTemp, FontSize, TEXTALIGN_LEFT, -1, 1, false);
```
Comment (def-) [inline src/game/client/components/menus_browser.cpp:447]: I think left aligned looks bad

**Comment by l-ouis** [comment]
What about this layout?
image


Testing the extremes:
image

I very slightly shrunk the space for the Server name for the extreme values to fit. Still, none of the currently online servers were truncated

**Comment by def-** [comment]
Maybe put the number a bit higher?

**Comment by l-ouis** [comment]
image
Flush with bottom (bumped a tiny bit upwards)

image
Centered

Personally like the flush with bottom more

**Comment by def-** [comment]
Agreed, seems fine.

**Comment by def-** [comment]
bors r+
Thanks! Looks nice

**Pull request merged**